cv_next/app/upload/page.tsx

```diff
@@ -47,7 +47,8 @@ const InputRow = ({
 };
 
 export default function Page() {
-  const [catagoryId, setCatagoryId] = useState<InputValues["catagoryId"]>(null);
+  const [catagoryId, setCatagoryId] =
+    useState<InputValues["cvCategories"]>(null);
```

Comment:
While you're at it, use constant instead of string for the field name (see relevant examples in other places in the code).
Also see other lines that needs such adjustment (53 and 54 for example).

Reply:
all I did was to change the name from catagoryId to cvCategories
and I do not understand the changes you are requesting

Reply:
and where can i find an example

Reply:
look for Cvkeys or something there

Reply:
i found exactly what he meant and apparently it not possible with an Interface such as InputValues

Reply:
Maybe it's because of the type that is mentioned there(in the supabase definitions)

Reply:
![image](https://github.com/user-attachments/assets/a5b6bcd0-334e-4f2c-93b5-092a77f84459)

Reply:
Wouldn't doing:
```ts
  const [catagoryId, setCatagoryId] = useState<number[]>([]);
  const [description, setDescription] = useState("");
  const [link, setLink] = useState("");
```

Solve the problem? No hardcoded strings or anything(obviously a bit more changes are needed for this but generally isn't it fine?